src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -238,6 +288,32 @@ private EmailWrapper generateFeedbackSessionEmailBaseForStudents(
         return email;
     }
     
+    private EmailWrapper generateFeedbackSessionEmailBaseForStudentSubmissionConfirmation(
+            CourseAttributes course, FeedbackSessionAttributes session, StudentAttributes student) {
+        String template = EmailTemplates.USER_FEEDBACK_SUBMISSION_CONFIRMATION;
+        String subject = EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject();
+        String submitUrl = Config.getAppUrl(Const.ActionURIs.STUDENT_FEEDBACK_SUBMISSION_EDIT_PAGE)
+                                 .withCourseId(course.getId())
+                                 .withSessionName(session.getFeedbackSessionName())
+                                 .withRegistrationKey(StringHelper.encrypt(student.key))
+                                 .withStudentEmail(student.email)
+                                 .toAbsoluteString();
+
+        String emailBody = Templates.populateTemplate(template,
+                "${userName}", student.name,
+                "${courseName}", course.getName(),
+                "${courseId}", course.getId(),
+                "${feedbackSessionName}", session.getFeedbackSessionName(),
+                "${deadline}", TimeHelper.formatTime12H(session.getEndTime()),
+                "${submitUrl}", submitUrl,
+                "${supportEmail}", Config.SUPPORT_EMAIL);
+        
+        EmailWrapper email = getEmptyEmailAddressedToEmail(student.email);
+        email.setSubject(String.format(subject, course.getName(), session.getFeedbackSessionName()));
+        email.setContent(emailBody);
+        return email;
+    }
```

Comment:
The two functions you defined are 80% similar. Extract the common parts to a common function, e.g
```java
private EmailWrapper generateSubmissionConfirmationEmail(
        CourseAttributes course, FeedbackSessionAttributes session, String submitUrl,
        String userName, String userEmail) {
    ...
}
```